README.md

```diff
@@ -371,6 +371,46 @@ in data.py.
         name: util-linux efitools
 ```
 
+## install_xcpng.py
+
+This script install (or upgrade) a VM into a XCP-ng host using a PXE server. You need to have the PXE server configured and running with the proper installer for each version. For more informations about PXE boot install you can read the XCP-ng documentation [here](https://xcp-ng.org/docs/install.html#pxe-boot-install).
+
+The script can display the help with `-h`:
+
+```
+usage: install_xcpng.py [-h] [--installer INSTALLER] host vm_uuid action version_xcpng
+
+positional arguments:
+  host                  The IP address or the name of the host where the VM is created.
+  vm_uuid               The UUID of an existing VM to use to create the new host.
+  action                install or upgrade.
+  version_xcpng         The version of xcp-ng we wish.
+
+options:
+  -h, --help            show this help message and exit
+  --installer INSTALLER
+                        URL of the installer on the PXE
+```
```

Comment:
You use sentences with capital letters and `.` for positional arguments, not here. ;)

Reply:
i don't understand. If it's about the 'INSTALLER' i don't know where it's coming from, it's not in the code.
i've just paste the return from the console.

Reply:
you need to change the help print by the script.
- `show this help message and exit` => `Show this help message and exit.`
- `URL of the installer on the PXE` => `URL of the installer on the PXE.`

Reply:
i don't understand.
i have this :
```
   parser = argparse.ArgumentParser()
    parser.add_argument("host", help="The IP address or the name of the host where the VM is created.")
    parser.add_argument("vm_uuid", help="The UUID of an existing VM to use to create the new host.")
    parser.add_argument("action", metavar='action', choices=['install', 'upgrade'], help="install or upgrade.")
    parser.add_argument("xcpng_version", help="The version of xcp-ng.")
    parser.add_argument("--installer", help="URL of the installer on the PXE")
    args = parser.parse_args()
```
The help is generated automaticly with thehelp=""
i don't know how to change that.

Reply:
You can edit the installer option description
As for the Help it can be changed with an optional argument IIRC.

Or you can remove all capital letters and `.` for consistency.
We do not have a preference.

Reply:
but i don't have any capital letter in the code :(